service/rtc/net.go

```diff
@@ -108,12 +108,15 @@ func createUDPConnsForAddr(log mlog.LoggerIFace, network, listenAddress string)
 			log.Info(fmt.Sprintf("rtc: server is listening on udp %s", listenAddress))
 		}
 
-		if err := udpConn.(*net.UDPConn).SetWriteBuffer(udpSocketBufferSize); err != nil {
-			log.Warn("rtc: failed to set udp send buffer", mlog.Err(err))
-		}
+		// Mac by default cannot set socket buffers > 8mb
+		if runtime.GOOS != "darwin" {
```

Comment:
Are you saying this works on Windows? :p

Reply:
What's a windows?

Reply:
Your future employer's OS :)